packages/react-components/react-components/src/Concepts/Icons/Icon.stories.mdx

```diff
@@ -25,8 +25,61 @@ You can import the sized icons in a similar way:
 import { AccessTime24Filled } from '@fluentui/react-icons';
 ```
 
+### List of available props
+
 Each icon also accepts `className` and `primaryFill` props for styling.
 
+| - Name -    | Description                                                                                                            | Default        |
+| ----------- | ---------------------------------------------------------------------------------------------------------------------- | -------------- |
+| bundleIcon  | Pass in the filled and unfilled version of an icon and this will combine them into one icon                            | -              |
+| className   | Used to style the icon                                                                                                 | -              |
+| filled      | Used to determine whether the filled or unfilled version of an icon is the default when `bundleIcon` is used `boolean` | -              |
+| primaryFill | Used to change the primary fill of the icon                                                                            | `currentColor` |
+| title       | Specifies the title attribute for the svg                                                                              | -              |
+
+### Example
+
+```tsx
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import {
+  AccessTimeFilled,
+  AccessTimeRegular,
+  bundleIcon,
+  iconFilledClassName,
+  iconRegularClassName,
+} from '@fluentui/react-icons';
+import { makeStyles } from '@griffel/react';
+
+const iconStyleProps: FluentIconsProps = {
+  primaryFill: 'purple',
+  className: 'iconClass',
+};
+
+const useIconStyles = makeStyles({
+  icon: {
+    ':hover': {
+      [`& .${iconFilledClassName}`]: {
+        display: 'none',
+      },
+      [`& .${iconRegularClassName}`]: {
+        display: 'inline',
+      },
+    },
+  },
+});
+
+const AccessTime = bundleIcon(AccessTimeFilled, AccessTimeRegular);
+const rootElement = document.getElementById('root');
+const styles = useIconStyles();
+ReactDOM.render(
+  <div className={styles.icon}>
+    <AccessTime aria-label="AccessTime" {...iconStyleProps} />
+  </div>,
+  rootElement,
+);
```

Comment:
- `useIconStyles` is a React hook a can be called inside React components only
- Let's avoid usage of React DOM in our examples as React 18 has [new API](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis) 
- I would be careful with `aria-label` as AFAIK we don't recommend to use it everywhere on icons

```suggestion
function App() {
  const styles = useIconStyles();

  return (
    <div className={styles.icon}>
      <AccessTime aria-label="An AccessTime icon" {...iconStyleProps} />
    </div>
  )
);
```